com.unity.netcode.gameobjects/Prototyping/NetworkTransform.cs

```diff
@@ -1,12 +1,15 @@
 using System;
+using System.Collections.Generic;
 using UnityEngine;
+using Random = UnityEngine.Random;
 
 namespace Unity.Netcode.Prototyping
 {
     /// <summary>
     /// A prototype component for syncing transforms
     /// </summary>
     [AddComponentMenu("Netcode/" + nameof(NetworkTransform))]
+    [DefaultExecutionOrder(1000)] // this is needed to catch the update time after the transform was updated by user scripts
```

Comment:
Magic number, we will need to document this. Also 1000 might be a bit low since this is pretty much a `almost always run latest` situation.

Reply:
that, + if we want to catch `transform` updates after user scripts, we should also put things into `LateUpdate()` I guess.

Reply:
Luke: do we have recommendations on what a good number for this would be? I usually see "10" "100" for other script orders.
Fatih: not if we have FixedUpdate, which could execute more than once before LateUpdate

Reply:
bump

Reply:
Could we bump it to something like `100'000`? I have seen people use 1000+ frequently even in some asset store packs and this is really a case of `we always want this last unless you know what you are doing`.

Reply:
done